metrics/expvar/factory.go

```diff
@@ -44,6 +46,10 @@ func (f *factory) Gauge(name string, help string) metrics.Gauge {
 	return xkit.NewGauge(f.factory.Gauge(name))
 }
 
-func (f *factory) Timer(name string, help string) metrics.Timer {
+func (f *factory) Timer(name string, help string, buckets []time.Duration) metrics.Timer {
 	return xkit.NewTimer(f.factory.Histogram(name))
 }
+
+func (f *factory) Histogram(name string, help string, buckets []float64) metrics.Histogram {
+	return xkit.NewHistogram(f.factory.Histogram(name))
```

Comment:
maybe a TODO to respect the buckets? Doesn't gokit allow buckets for histograms?

Reply:
Added.